relay-general/src/store/mod.rs

```diff
@@ -137,3 +140,25 @@ impl<'a> Processor for StoreProcessor<'a> {
         Ok(())
     }
 }
+
+pub fn light_normalize<'a>(
+    event: &mut Annotated<Event>,
+    client_ip: Option<&IpAddr>,
+    user_agent: Option<&'a str>,
+    received_at: Option<DateTime<Utc>>,
+    max_secs_in_past: Option<i64>,
+    max_secs_in_future: Option<i64>,
+    breakdowns_config: Option<BreakdownsConfig>,
```

Comment:
These are now enough parameters to warrant some form of config struct like we have in the store normalizer.